### ISSUE TITLE
Can't get scripts working: No module named zabbix_api

### ISSUE THREAD

**Issue by Blasterdick** [gnetsman/zabbix_api#5]
Traceback (most recent call last):
  File "./zabbix_host_del.py", line 3, in <module>
    from zabbix_api.zabbix_methods import *
  File "/usr/local/lib/python2.7/site-packages/zabbix_api/zabbix_methods.py", line 4, in <module>
    from zabbix_api.zabbix_api import ZabbixAPI
ImportError: No module named zabbix_api

Python 2.7.10

>>> import zabbix_api
>>> dir(zabbix_api)
['APITimeout', 'Already_Exists', 'InvalidProtoError', 'ZabbixAPI', 'ZabbixAPIException', 'ZabbixAPISubClass', '__builtins__', '__doc__', '__file__', '__logger', '__name__', '__package__', 'base64', 'checkauth', 'default_log_handler', 'deque', 'dojson', 'hashlib', 'json', 'logging', 're', 'socket', 'ssl', 'string', 'sys', 'urllib2']

**Comment by mbadley** [comment]
./zabbix_host_del.py --help
Traceback (most recent call last):
  File "./zabbix_host_del.py", line 3, in <module>
    from zabbix_api.zabbix_methods import *
ImportError: No module named zabbix_methods

**Comment by Hubbitus** [comment]
I have similar error:
```
Traceback (most recent call last):
  File "/usr/bin/zabbix_tool", line 907, in <module>
    tool.login(**options)      # Cheating!
  File "/usr/bin/zabbix_tool", line 103, in login
    self.z = zabbix_api.ZabbixAPI(server=url, timeout=timeout, validate_certs=validate_certs)
AttributeError: 'module' object has no attribute 'ZabbixAPI'
```

Is that project alive?

**Comment by Blasterdick** [comment]
@Hubbitus nope, it doesn't seems to be)) and I gave up to make that thing work, as I'm a newbie in programming. Maybe you'll give it a try?
